src/D2L.Security.OAuth2/Validation/AccessTokens/AccessTokenValidator.cs

```diff
@@ -84,10 +84,12 @@ string token
 					token,
 					validationParameters,
 					out securityToken
-        );
+        		);
 				accessToken = new AccessToken( ( JwtSecurityToken )securityToken );
 			} catch( SecurityTokenExpiredException e ) {
 				throw new ExpiredTokenException( e );
+			} catch( SecurityTokenNotYetValidException e ) {
+				throw new ValidationException( "Token was from the future (nbf)", e );

```

Comment:
nit: expired token exception message is in present tense. _The access token is expired_.

Reply:
Good call that read pretty weird